src/app/chat/[id]/page.tsx

```diff
@@ -0,0 +1,76 @@
+"use client";
+
+import { useParams, useRouter } from "next/navigation";
+import { Button } from "@/components/ui/button";
+import { useEffect, useState } from "react";
+import DialogTemplate from "@/components/ui/dialog";
+import { Slot } from "@radix-ui/react-slot";
+
+
+export default function Chat() {
+  const router = useRouter();
+
+  const { id } = useParams(); 
+  
+  const [actionsRequired, setActionsRequired] = useState(["action1"]);
+  
+  const [pastRequests, setPastRequests] = useState(["request1"]);
+  
+
+
+  
+  useEffect(() => {
+
+  }, [id])
+
+  const getPastRequest = async (chatId : number) => {
+    try {
+
+    } catch {
+
+    } finally {
+
+    }
+  }
+
+  return (
+
+      <main className="flex justify-center grid grid-cols-4 max-h-full">
+        <div className="flex col-span-1">
+        </div>
+
+        <div className="flex col-span-2 h-screen bg-gray-300 grid grid-rows-12 p-4 items-end">
```

Comment:
```suggestion
      <main className="justify-center grid grid-cols-4 max-h-full">
        <div className="flex col-span-1">
        </div>

        <div className="col-span-2 h-screen bg-gray-300 grid grid-rows-12 p-4 items-end">
```

flex is overwritten by grid and has no effect